azure-pipelines.yml

```diff
@@ -14,7 +14,8 @@ variables:
                         -DWITH_OPENCV=1 \ 
                         -DWITH_EXAMPLES=1 \
                         -DWITH_DOC=0 \
-                        -DWITH_OPEN3D=0"
+                        -DWITH_OPEN3D=0 \ 
+                        -DCI_BUILD=1"
```

Comment:
Please remove the above line "-DUSE_DEPTH_COMPUTE_STUBS=1 \"
The CI_BUILD will enable it anyway.

Reply:
Fixed.